doc/Deploy_Manual.4D03.md

```diff
@@ -265,6 +265,7 @@ D03 board supports booting via SAS, USB and SATA by default. The UEFI will direc
             }
        ```
        Note:<br>
+        *  From the formal version of V2.2 and later versions value of console is `ttys0115200`, if you use V2.2 and previous versions value of console is `ttys1,115200`
```

Comment:
This description is incorrect. D03 with different SW release after V2.2 release may use ttyS0 or ttyS1.
And ttys0115200 / ttys1,115200 --> Both are incorrect.